.eslintrc.json

```diff
@@ -136,6 +136,14 @@
                 "react/jsx-key": "off",
                 "no-console": "off"
             }
+        },
+        {
+            "files": ["packages/react-integration/demo-app-ts/**/*"],
+            "rules": {
+                "patternfly-react/no-anonymous-functions": "off",
```

Comment:
This linting rule gives false positives for class components that already have a displayName, so I've turned this off. For example:

```ts
class MyComponent extends React.Component {
  static displayName = "MyComponent"; // It doesn't like this.
}
```